tfsec-to-trivy-migration-guide.md

```diff
@@ -0,0 +1,48 @@
+# Moving towards configuration scanning with Trivy
+Overtime we've taken [trivy][trivy] to be the go-to scanning tool for a vareity of things. This also includes terraform scanning.
+
+This section describes some differences between Trivy and tfsec.
+
+| Feature              | Trivy                                                  | tfsec                |
+|----------------------|--------------------------------------------------------|----------------------|
+| Policy Distribution | Embedded and Updated via Registry                      | Embedded             |
```

Comment:
can we describe this from the use value perspective? embedded vs registry is technically correct, but why should the user care